wp-content/themes/reconasia/assets/_scss/blocks/post/_publications.scss

```diff
@@ -1,11 +1,27 @@
 @use '../abstracts' as *;
 
 .wp-block-lazyblock-post-publication {
+  --margin-text-side: auto;
+  --margin-gutter-side: auto;
+  max-width: 240px !important; // Override the alignment max-width
+
   .csis-block--publication {
     display: grid;
     grid-column-gap: rem(16);
     grid-row-gap: 0;
     grid-template-columns: min-content auto;
+
+    &:hover {
```

Comment:
TIL about [:focus-within](https://developer.mozilla.org/en-US/docs/Web/CSS/:focus-within), which actually let me get close to what the mockups wanted. I also added an outline to the individual element that is selected, just to avoid confusion. The way this block is set up is that the individual pieces are linked instead of a single big link, and I couldn't find a reliable way to skip over them, so I opted to add an outline to the individually selected item in addition to the other changes in the mockup.